@here/harp-mapview/lib/MapView.ts

```diff
@@ -1281,8 +1295,19 @@ export class MapView extends THREE.EventDispatcher {
         this.m_textElementsRenderer.clearRenderStates();
         this.m_renderer.dispose();
         this.m_imageCache.clear();
+        this.m_tileGeometryManager.clear();
 
         this.m_movementDetector.dispose();
+
+        // Remove all event handlers.
+        super.dispose();
```

Comment:
We are still missing the termination of the workers.
I'm actually not sure how to do it b/c `ConcurrentDecoderFacade` and `ConcurrentTilerFacade` are using some singleton-ish pattern.
If we have multiple `MapView`s, we need to do some reference counting to only terminate the workers if no `MapView` needs them anymore.

Reply:
The WorkerBasedDecoder is disposed of, and the decoders are stopped, no events from them should arrive at MapView after that. I guess for all normal use cases this is good enough.

Reply:
I see, there is some reference counting in `WorkerBasedDecoder` and `WorkerBasedTiler`.

Nevertheless I cannot see that we call dispose on the `ConcurrentTilerFacade` anywhere.

Reply:
I will add that.